style.css

```
@@ -0,0 +1,267 @@
+@font-face {
+  font-family: Pretendard;
+  src: url("https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css");
+}
+
+:root {
+  --primary-100: #3692ff;
+  --primary-200: #1967d6;
+  --primary-300: #1251aa;
+  --secondary-100: #f9fafb;
+  --secondary-200: #f3f4f6;
+  --secondary-300: #e5e7eb;
+  --secondary-400: #9ca3af;
+  --secondary-500: #6b7280;
+  --secondary-600: #4b5563;
+  --secondary-700: #374151;
+  --secondary-800: #1f2937;
+  --secondary-900: #111827;
+  --error-red: #f74747;
+  --background-color: #ffffff;
+  --background-top-bottom-color: #cfe5ff;
+
+}
+
+/* P 태그의 폰트 사이즈 기준 설정*/
+html {
+  font-size: 24px;
+}
+
+@media(max-width: 1024px) {
+  html {
+    font-size: 18px;
+  }
+}
+
+@media(max-width: 768px) {
+  html {
+    font-size: 16px;
+  }
+}
+
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: var(--background-color);
+  color: var(--secondary-700);
+  font-family: "Pretendard", system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

P2:
필요한 속성, 값만 있으면 코드 가독성이 더 좋아질 것 같습니다.

```suggestion
  font-family: "Pretendard", sans-serif;
```